habitat_sim/agent/controls/pyrobot_noisy_controls.py

```diff
@@ -26,14 +26,13 @@
 from habitat_sim.registry import registry
 
 
-@attr.s(auto_attribs=True)
```

Comment:
Why no more `attr.s`?  This was nice for debugging since I could just print the object.  If you want to get rid of `__init__`, you can add `init=False` to the decorator.

Reply:
Yeah, in the future we may want to consider using dataclasses since they are part of the standard python library in 3.7>=